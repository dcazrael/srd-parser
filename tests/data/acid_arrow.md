#### Acid Arrow

Level 2, Evocation, [Wizard]

**Casting Time**: Action
**Range**: 90 feet
**Components**: V, S, M (powdered rhubarb leaf)
**Duration**: Instantaneous

A shimmering green arrow streaks toward a target within range and bursts in a spray of acid. Make a ranged spell attack against the target. On a hit, the target takes 4d4 Acid damage and 2d4 Acid damage at the end of its next turn. On a miss, the arrow splashes the target with acid for half as much of the initial damage only.

**Using a Higher-Level Spell Slot**. The damage increases by 1d4 for each spell slot level above 2.