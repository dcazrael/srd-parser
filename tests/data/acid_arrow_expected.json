{
  "name": "Acid Arrow",
  "level": 2,
  "school": "Evocation",
  "classes": ["Wizard"],
  "casting_time": "Action",
  "range": "90 feet",
  "components": "V, S, M (powdered rhubarb leaf)",
  "duration": "Instantaneous",
  "description": "A shimmering green arrow streaks toward a target within range and bursts in a spray of acid. Make a ranged spell attack against the target. On a hit, the target takes 4d4 Acid damage and 2d4 Acid damage at the end of its next turn. On a miss, the arrow splashes the target with acid for half as much of the initial damage only.",
  "higher_levels": "The damage increases by 1d4 for each spell slot level above 2.",
  "subsections": []
}
